### DISCUSSION TITLE
Provide separate settings for default view branch vs. default PR base branch

### DISCUSSION THREAD

**Discussion by EliteMasterEric** [community/community#5705] [Pull Requests]
Often, in GitFlow-style setups, the main branch (which most users are encouraged to use) is separate from the development branch users are encouraged to make PRs to.

Having the option to make a different branch from main be the default PR branch would be excellent.

Additionally, I'd like to see an option to prevent outside pull requests to certain branches. For example, a fork's pull request can only be made towards my develop branch, but my develop branch can go to master.

**Comment by rmi22186** [comment]
I found this discussion after being directed here from https://github.com/isaacs/github/issues/1401, where there was much more engagement with the issue:
![image](https://user-images.githubusercontent.com/5377436/134524117-23609ea5-577b-4a77-904d-1a4b90b12bb6.png)

I imagine this is a feature that would be leveraged by a high percentage of repositories. We want other developers to see what code is live, which is the master/main branch.  However, defaulting to PR-ing into the development branch is preferred given the reasons @kfranqueiro listed.  100x BUMP this!

**Comment by DaniBodor** [comment]
I'm curious why this issue hasn't gained more traction. The issue @rmi22186 mentions has been archived, and this one seems largely ignored :(
